dashboard/src/components/DeploymentForm/DeploymentForm.tsx

```diff
@@ -193,11 +197,21 @@ class DeploymentForm extends React.Component<IDeploymentFormProps, IDeploymentFo
             </div>
             <div className="col-8">
               <div>
-                <label htmlFor="releaseName">Name</label>
+                <label htmlFor="helmCRDReleaseName">Name</label>
                 <input
-                  id="releaseName"
+                  id="helmCRDReleaseName"
+                  onChange={this.handleHelmReleaseNameChange}
+                  value={this.state.helmCRDReleaseName}
+                  required={true}
+                  disabled={hr ? true : false}
+                />
+              </div>
+              <div>
+                <label htmlFor="tillerReleaseName">Release Name (Global)</label>
```

Comment:
Yeah, I agree it could be confusing. I would do 1. (or any other method that checks if a release already exists) more than 2. since this is something more specific to kubeapps.

Reply:
Yeah, I think you're right, it doesn't make sense to impose the restriction onto Helm CRD itself.